cmd/kubeadm/app/cmd/init.go

```diff
@@ -244,47 +248,54 @@ func (i *Init) Run(out io.Writer) error {
 		return err
 	}
 
+	// PHASE 4: Mark the master with the right label/taint
 	if err := markmasterphase.MarkMaster(client, i.cfg.NodeName); err != nil {
 		return err
 	}
 
-	// PHASE 4: Set up the bootstrap tokens
+	// PHASE 5: Set up the node bootstrap tokens
```

Comment:
Minor nit: it might be easier to maintain these comments if we dropped the numbering.